applications/GeoMechanicsApplication/tests/test_line_interface_elements.py

```diff
@@ -135,5 +135,32 @@ def test_multi_stage_3_plus_3_line_interface_element_with_neumann_conditions(sel
 
         os.chdir(initial_cwd)
 
+    def assert_results_of_multi_stage_test_with_reset_displacement(self, stage, displacement_vector):
+        expected_displacement_vectors = [[0.0, 0.0, 0.0]] * 3  # the first three nodes have been fixed
+        expected_displacement_vectors += [displacement_vector] * 3  # the last three nodes have prescribed non-zero displacements
+        self.assertVectorsAlmostEqual(test_helper.get_displacement(stage), expected_displacement_vectors)
+
+        expected_normal_relative_displacement = displacement_vector[1]
+        expected_tangential_relative_displacement = displacement_vector[0]
+        expected_relative_displacement_vectors = [[expected_normal_relative_displacement, expected_tangential_relative_displacement]] * 3
+        self.assertVectorsAlmostEqual(test_helper.get_on_integration_points(stage, Kratos.STRAIN)[0],
+                                      expected_relative_displacement_vectors)
+
+    def test_multi_stage_3_plus_3_line_interface_element_with_neumann_conditions_and_reset_displacements(self):
+        file_path = test_helper.get_file_path(os.path.join('line_interface_elements', 'Neumann_multi_stage_reset_displacements'))
+
+        initial_cwd = os.getcwd()
```

Comment:
Initial current working directory? Behaviour reads like I remember where I am now and will return here when finished wandering. initial_directory is more clear to me as variable name.

Reply:
No objections there!